neptune/man/neptune_upload.Rd

```diff
@@ -0,0 +1,46 @@
+\name{neptune_upload}
+\alias{neptune_upload}
+\title{
+Uploads provided file under specified field path
+}
+\description{
+Uploads provided file under specified field path
+}
+\usage{
+neptune_upload(x, value)
+}
+\arguments{
+  \item{x}{
+Neptune handler
+}
+  \item{value}{
+string - Path to the file to be uploaded
```

Comment:
or File value object ?

Reply:
I omitted it on purpose, cos you don't really have a compatible file obj on R's side

Reply:
But there are examples like `neptune_upload(run['evaluation/predictions'], neptune_file_as_html(df))` so someone would ask what is the type of result of `neptune_file_as_html` + It was already named as `an HTML File value object` :wink:

Reply:
completly forgot 'bout that lol